package.json

```diff
@@ -26,21 +26,22 @@
   },
   "license": "Apache-2.0",
   "author": "Kaitai team (https://github.com/orgs/kaitai-io/people) and Ilmari Heikkinen",
+  "type": "commonjs",
   "main": "index.js",
   "types": "index.d.ts",
   "files": [
-    "KaitaiStream.js",
-    "KaitaiStream.d.ts",
-    "index.d.ts"
+    "*.js",
+    "*.d.ts"
```

Comment:
Actually, I don't think this the best idea - it makes it possible to accidentally publish some other `.js` file. Let's keep it explicit.